doc/security.md

```
@@ -0,0 +1,100 @@
+
+Cloudflare Access provides an authentication layer that uses your existing identity providers, such as Google or GitHub accounts, to control access to your applications. When using Cloudflare Access for authentication, you can configure BirdNET-Go to trust traffic coming through the Cloudflare tunnel. The system authenticates requests by validating the `Cf-Access-Jwt-Assertion` header containing a JWT token from Cloudflare.
+
+To add even more security, you can also require that the Cloudflare Team Domain Name and Policy audience are valid in the JWT token. Enable these by defining them in the `config.yaml` file:
+
+```yaml
+security:
+  allowcloudflarebypass:
+          enabled: true
+          teamdomain: "your-subdomain-of-cloudflareaccess.com"
+          audience: "your-policy-auddience"
+```	
```

Markdownlint

Column: 4
Hard tabs

(MD010, no-hard-tabs)